### PR TITLE
Updates for building host tools against the latest TAO 2.2a.

### PR DESCRIPTION
**Pull request by ocielliottc**

### CODE REVIEW

File: tools/scripts/create_universal_linux_dds_idl_binaries.sh [hunk 1]
```diff
@@ -84,18 +84,21 @@ threads = 0
 debug = 0
 static_stdlibs = 1
 dynamic_loader = 0
+tao_idl_fe_gen = 1
```
Comment (mitza-oci) [inline tools/scripts/create_universal_linux_dds_idl_binaries.sh:87]: Could it work without this?  The release archive should already have the generated files.

Comment (ocielliottc) [reply tools/scripts/create_universal_linux_dds_idl_binaries.sh:87]: It didn't build without it.

Comment (jwillemsen) [reply tools/scripts/create_universal_linux_dds_idl_binaries.sh:87]: We don't need that option in our host builds using doc_group TAO 3

Comment (mitza-oci) [reply tools/scripts/create_universal_linux_dds_idl_binaries.sh:87]: @jwillemsen are you using this script?  Whatever the underlying issue is, it seems to be unique to this script or the environment it's run in.

I was able to build successfully without `tao_idl_fe_gen` on an Ubuntu VM.  I only changed buildbits to 64.

Comment (jwillemsen) [reply tools/scripts/create_universal_linux_dds_idl_binaries.sh:87]: Not with this script, but we do something similar, create a static host tree compilation of just the minimal parts we need on a host system, never ever used `tao_idl_fe_gen` on any build

**Review by mitza-oci — APPROVED** [review]

**Pull request merged**